be1-go/hub/standard_hub/mod_test.go

```diff
@@ -1793,3 +1952,38 @@ func (f *fakeSocket) ID() string {
 func (f *fakeSocket) Type() socket.SocketType {
 	return socket.ClientSocketType
 }
+
+func IntPointer(i int) *int {
```

Comment:
is this used anywhere? also is it public on purpose?

Reply:
Good catch, I was supposed to remove it.